html/donate/aa.html

```diff
@@ -87,7 +87,7 @@
         z-index: 0;
       }
     </style>
-    <script src="https://static.openfoodfacts.org/js/dist/jquery.js"></script>
+    <script src="[% static_subdomain %]/js/dist/jquery.js"></script>
```

Comment:
The static pages in `html/donate` cannot use the `Template::Toolkit` variables. They probably need to stay as-is